frontend/src/forms/CreateAccountFormDialog.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+
+// Redux
+import { useDispatch } from 'react-redux';
+import { postUserRequest } from '../state/user/userActions';
+
+// Form
+import { TextField } from 'mui-rff';
+import FormDialog from '../common/components/FormDialog';
+import createAccountSchema from './CreateAccountFormDialog.schema';
+
+// Util
+import queryString from 'query-string'
```

Comment:
too lazy to make my method to convert from key values to query string param. found this little library and embraced the power of javascript